src/Microsoft.VisualBasic.Core/tests/StringsTests.cs

```diff
@@ -143,6 +143,7 @@ public void Asc_Chr_Invariant(int charCode, int expected)
         [InlineData(0x8141, 0x8141)]
         [InlineData(0xC8FE, 0xC8FE)]
         [InlineData(0xFFFF, 0xFF)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
What was the failure here?

Reply:
```
Assert.Equal() Failure
  Expected: 255
  Actual:   63
```
and
```
Assert.Equal() Failure
  Expected: 51454
  Actual:   63
```
and
```
Assert.Equal() Failure
          Expected: 33089
          Actual:   63
```

Reply:
@cston, if that's expected, I'll add a comment as to why.  If it's not, I can open an issue and switch to using ActiveIssue for it.

Reply:
I'll need to investigate if that's the correct behavior. Feel free to open an issue.

Reply:
Thanks. Opened https://github.com/dotnet/corefx/issues/39888.